### PR TITLE
Adding Sentinel imagery clipped to coast

### PR DESCRIPTION
**Pull request by josephramsay**
Fixes: #  Ragged boundary and inconsistent colouring between sentinel, gebco and aerial imagery 

### Change Description:

Sentinel imagery clipped to coast line shape file

### CODE REVIEW

File: packages/lambda-xyz/src/cli/imagery/sentinel.ts [hunk 1]
```diff
@@ -1,10 +1,74 @@
...
+import { MosaicCog } from './mosaic.cog';
 
 MosaicCog.create({
-    id: '01E4PN40AEH10EH128AG28YSDM',
-    name: 'new_zealand_sentinel_2018-19_10m',
-    quadKeys: ['31'],
+    id: '01E6JDD1C6RMH29Q8Z97AEE79S',
+    name: 'sentinel',
+    quadKeys: [
+        '3113332',
```
Comment (blacha) [inline packages/lambda-xyz/src/cli/imagery/sentinel.ts:8]: This is way too many quad keys for this imagery.

It should be at most about 5?

File: packages/lambda-xyz/src/cli/imagery/sentinel.ts [hunk 2]
```diff
@@ -1,10 +1,74 @@
-import { MosaicCog } from './mosaic.cog';
 import { EPSG } from '@basemaps/geo';
+import { MosaicCog } from './mosaic.cog';
 
 MosaicCog.create({
-    id: '01E4PN40AEH10EH128AG28YSDM',
-    name: 'new_zealand_sentinel_2018-19_10m',
-    quadKeys: ['31'],
+    id: '01E6JDD1C6RMH29Q8Z97AEE79S',
+    name: 'sentinel',
```
Comment (blacha) [inline packages/lambda-xyz/src/cli/imagery/sentinel.ts:6]: We should be keeping the name from what it is in the datalake `new_zealand_sentinel_2018-19_10m`

Comment (palmerj) [reply packages/lambda-xyz/src/cli/imagery/sentinel.ts:6]: Note the new data lake structure has changed: https://confluence.linz.govt.nz/display/GEOD/File+and+Folder+Structure+in+the+Raster+Data+Store

Differences what you have now is the year ranges will include the full year string and the data type will be appended to the end (e.g RGBA)

Comment (josephramsay) [reply packages/lambda-xyz/src/cli/imagery/sentinel.ts]: Changed back to new_zealand_sentinel_2018-19_10m

File: packages/lambda-xyz/src/cli/imagery/sentinel.ts (hunk 1 above)
Comment (josephramsay) [reply packages/lambda-xyz/src/cli/imagery/sentinel.ts]: Set max-cogs to 5, returned 3 quadkeys

**Comment by josephramsay** [comment]
Reprocessed with 3 quadkeys, missed cutline. Will redo

**Pull request merged**